index.html

```diff
@@ -12,45 +12,77 @@
     <link rel="stylesheet" href="stylesheet.css">
     <link rel="icon" type="image/png" sizes="32x32" href="Assets/Favicon/favicon-connectdevelop.ico">
     <meta name="theme-color" content="#ffffff">
+
```

Comment:
Czemu zostawiasz tyle wolnych linijek? Dziwnie to wygląda w git diff i sprawia wrażenie, jakbyś dodał znacznie poważniejsze zmiany niż dodałeś naprawdę.